src/localize-router.pipe.ts

```diff
@@ -37,7 +34,7 @@ export class LocalizeRouterPipe implements PipeTransform {
     if (!query || query.length === 0 || !this.localize.parser.currentLang) {
       return query;
     }
-    if (equals(query, this.lastKey) && equals(this.lastLanguage, this.localize.parser.currentLang)) {
+    if (!this.markForTransform && query === this.lastKey && this.lastLanguage === this.localize.parser.currentLang) {
```

Comment:
the reason why I remove the `equals` was that a ref compare will be way faster, especially if its invoked too many times, and if the reference changes then simply might as well recompute it